server/api/scenarios.post.ts

```diff
@@ -0,0 +1,17 @@
+import { runScenario } from "@/server/handlers/scenarios";
+import { defineRApiEventHandler } from "~/server/utils/defineRApiEventHandler";
+import type { NewScenarioResponse } from "@/types/apiResponseTypes";
+import type { ParameterDict } from "@/types/apiRequestTypes";
+
+export default defineRApiEventHandler(
+  async (event): Promise<NewScenarioResponse> => {
+    const query = getQuery(event);
+
+    const modelParameters = JSON.parse(query.parameters as string) as ParameterDict;
+
+    // Delegate to runScenario so that the logic can be unit-tested.
```

Comment:
dont need this comment

Reply:
I've moved this comment - which was getting duplicated for each api endpoint - into a new .md "API design decisions.md"